.specify/scripts/bash/common.sh

```
@@ -28,24 +28,27 @@ get_current_branch() {
-                local dirname=$(basename "$dir")
-                if [[ "$dirname" =~ ^([0-9]+)- ]]; then
-                    local number=${BASH_REMATCH[1]}
-                    number=$((10#$number))
-                    if [[ "$number" -gt "$highest" ]]; then
-                        highest=$number
-                        latest_feature=$dirname
+        for type_dir in "$specs_base"/*/; do
+            [[ -d "$type_dir" ]] || continue
+            for dir in "$type_dir"*/; do
+                if [[ -d "$dir" ]]; then
+                    local dirname=$(basename "$dir")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)